modules/installation-cloudformation-vpc-localzone.adoc

```diff
@@ -0,0 +1,379 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-localzone.adoc
+
+[id="installation-cloudformation-vpc-localzone_{context}"]
+= CloudFormation template for the VPC that uses local zones
+
+You can use the following CloudFormation template to deploy the VPC that
+you need for your {product-title} cluster that uses local zones.
+
+.CloudFormation template for the VPC
+[%collapsible]
+====
+[source,yaml]
+----
+AWSTemplateFormatVersion: 2010-09-09
+Description: Template for Best Practice VPC with 1-3 AZs
+
+Parameters:
+  ClusterName:
+    Type: String
+    Description: ClusterName used to prefix resource names
+  ClusterInfraId:
+    Type: String
+    Default: "unmanaged"
+    Description: ClusterInfraId used to tag required resources
```

Comment:
I need to review the `unmanaged` initial value for `ClusterInfraId`. The InfraID is a chicken-and-egg problem where we need the VPC created to place the subnets in install-config to get the generated InfraId. In the original article, I place an option to patch this. Let's keep this thread open until I review if it is a must or not.

Reply:
Let us know the best approach!

Reply:
The documentation of ["Requirements for using your VPC"](https://docs.openshift.com/container-platform/4.11/installing/installing_aws/installing-aws-vpc.html#installation-custom-aws-vpc-requirements_installing-aws-vpc) when installing a cluster in existing VPC, says `The VPC must not use the kubernetes.io/cluster/.*: owned tag.`. This is in consensus what we are doing here. You can skip this comment.

Reply:
`ClusterInfraId` is not required while creating VPC like what [original VPC cloudformation template](https://docs.openshift.com/container-platform/4.11/installing/installing_aws/installing-aws-user-infra.html#installation-cloudformation-vpc_installing-aws-user-infra) does, tag `kubernetes.io/cluster/<infra-id>: shared` will be added by installer automatically

Reply:
@mtulio @yunjiang29 Does this mean I should delete the four lines of "ClusterInfraId:" or leave them alone?

Reply:
@GroceryBoyJr you can remove this parameter, please read my comment below on the default value that should be defined when removing this.

https://github.com/openshift/openshift-docs/pull/54535/files#r1070098476

cc @yunjiang29